com.wetgenes.gamecake.desktop

```
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=GameCake
+Comment=GameCake is a cross platform game engine, a build of Lua(JIT) with embedded libraries.
+Exec="/app/bin/gamecake -lfun"
```

```suggestion
Exec=gamecake -lfun
```

It's not a path
and it's not quoted.